functions/go/gatekeeper-validate/README.md

```diff
@@ -0,0 +1,43 @@
+# gatekeeper-validate
+
+### Overview
+
+<!--mdtogo:Short-->
+
+Validate the KRM resources using the policy controller.
```

Comment:
"policy controller" is the admission controller and reserved for proprietary Google product. This is the OSS Gatekeeper constraint framework (Hence the previous discussion around name of the image). @maxsmythe Please check for correct usage.

Reply:
I'd say "using Gatekeeper constraints"